### ISSUE TITLE
Add possibility to set max header size for requests

### ISSUE THREAD

**Issue by markusmuellerusi** [DependencyTrack/dependency-track#3388]
### Current Behavior

Referring to discussion https://github.com/DependencyTrack/dependency-track/discussions/3384

It's currently not possible to set the max header size for embedded Jetty server.
In class EmbeddedJettyServer public method main (package alpine.embedded) there's no way to to set this configuration property by CLI argument or config file. 

### Proposed Behavior

Proposal:

---
 public static void main(String[] args) throws Exception {
---
CliArgs cliArgs = new CliArgs(args);
---
Server server = new Server();
        HttpConfiguration httpConfig = new HttpConfiguration();
        httpConfig.addCustomizer(new ForwardedRequestCustomizer());
        HttpConnectionFactory connectionFactory = new HttpConnectionFactory(httpConfig);
        ServerConnector connector = new ServerConnector(server, new ConnectionFactory[]{connectionFactory});
        connector.setHost(host);
        connector.setPort(port);
---

I would recommend to add a CLI arg and pass it to method httpConfig.setRequestHeaderSize(cliArgs.switchIntegerValue("-maxRequestHeaderSize", 8192));

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this enhancement was already requested

**Comment by markusmuellerusi** [comment]
Hi, is somebody currently working on depedency-track project? A feedback would be wellcome. Thanks

**Comment by stevespringett** [comment]
Dependency-Track does not directly embed Jetty. It uses Java Alpine-Executable-War, specifically https://github.com/stevespringett/Alpine/blob/master/alpine-executable-war/src/main/java/alpine/embedded/EmbeddedJettyServer.java

**Comment by markusmuellerusi** [comment]
Right, the problem is, there are many web apps hosted in same domain. Dependency-Track is just hosted in a virtual directory (sub path) on that server. Each web application uses a large number of OpenId connect groups, the user can be member of. So the default size exceeds very often and the user will get an 431 http status, if he/she tries to login. @stevespringett I know that this is dependend on alpine. That*s the reason I cannot fix it in Dependency-Track directly and need a kind of config to tell alpine handle large headers. Thanks.

**Comment by markusmuellerusi** [comment]
Please add the possibility to pass cli args to configure jetty, not only proxy host and port.
